bunner-macroquad/.github/workflows/crosscompile.yml

```diff
@@ -0,0 +1,50 @@
+name: Cross-compile
```

Comment:
If I understand correctly, the `bunner-macroquad/.github/workflows/*`files in the PR are non-functional, as GH Actions only looks in `<root>/.github/workflows` (see [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions)).

Can you remove these files from the PR, and open an issue linking to them in your own repository (https://github.com/ollej/rust-bunner-macroquad/tree/main/.github/workflows)? Regardless of the fact that they're used or not in this repository, I'd like to discuss separately about their integration - I think it's a good idea, but better to shape it as a separate topic.

Reply:
Yes, this was a mistake to include it with n the PR. I will remove them.